source/features/tags-dropdown.tsx

```diff
@@ -37,15 +37,27 @@ function init(): false | void {
 	select('.rgh-tags-dropdown')!.addEventListener('remote-input-success', updateLinksToTag);
 }
 
+function onFragmentLoaded(): void {
+	changeTabToTags();
+	removeTabsAndHeader();
+}
+
 // We're reusing the Branch/Tag selector from the repo's Code tab, so we need to update a few things
 function changeTabToTags(): void {
-	// Change the tab to "Tags"
-	select('.rgh-tags-dropdown .select-menu-tab:last-child button')!.click();
+	// Select "Tags" tab
+	select('.rgh-tags-dropdown .SelectMenu-tab:last-child')!.click();
+}
+
+function removeTabsAndHeader(): void {
+	// Remove header
+	select('.rgh-tags-dropdown .SelectMenu-header')!.remove();
+	// Remove "tabs"
+	select('.rgh-tags-dropdown .SelectMenu-tabs')!.remove();
```

Comment:
Can this be done via CSS instead?

Reply:
🤦‍♂ I completely forgot about it. CSS is already there, just need to fix the selectors. Thanks for pointing out!